internal/stdin/stdin.go

```diff
@@ -19,7 +19,7 @@ func Read() (string, error) {
 
 	for {
 		r, _, err := reader.ReadRune()
-		if err != nil && err == io.EOF {
+		if err != nil || err == io.EOF {
```

Comment:
Correct me if I'm wrong but we probably should `continue` rather than `break` if `err != nil` and `break` if `err == io.EOF`?